apps/condo/domains/contact/components/ContactsEditor.tsx

```diff
@@ -0,0 +1,556 @@
+import React, { useCallback, useState, useMemo, useRef, useEffect } from 'react'
+import { Contact } from '@condo/domains/contact/utils/clientSchema'
+import { Contact as TContact } from '@condo/domains/contact/schema'
+import { Input, Row, Col, Select, Typography, Radio, Form, FormInstance, Skeleton } from 'antd'
+import { Button } from '@condo/domains/common/components/Button'
+import { PhoneInput } from '@condo/domains/common/components/PhoneInput'
+import { BaseSearchInput } from '@condo/domains/common/components/BaseSearchInput'
+import { green, grey } from '@ant-design/colors'
+import { OptionProps } from 'antd/lib/mentions'
+import { useIntl } from '@core/next/intl'
+import { PlusCircleFilled, MinusCircleFilled } from '@ant-design/icons'
+import { find, get, pick, has } from 'lodash'
+import { useTicketValidations } from '@condo/domains/ticket/components/BaseTicketForm/useTicketValidations'
+import styled from '@emotion/styled'
+import { useApolloClient } from '@core/next/apollo'
+import { searchContacts } from '@condo/domains/ticket/utils/clientSchema/search'
+import { useOrganization } from '@core/next/organization'
+import { ErrorsWrapper } from '@condo/domains/ticket/components/BaseTicketForm/ErrorsContainer'
+
+interface ILabelsProps {
+    left: React.ReactNode,
+    right?: React.ReactNode,
+}
+
+/**
+ * Displays validation error, but hides form input
+ */
```

Comment:
redurant comment

Reply:
I don't think so, because in the comment I mentioned what is missing in the code.